app/src/main/java/org/oppia/app/drawer/NavigationDrawerItem.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.drawer
+
+import org.oppia.app.R
+
+/** Enum to store the menu of [NavigationDrawerFragment] and get menu by id. */
+enum class NavigationDrawerItem(val value: Int) {
+  HOME(R.id.nav_home), HELP(R.id.nav_help);
+
+  companion object {
+    fun valueFromInt(v: Int): NavigationDrawerItem {
+      var item = HOME
+      when (v) {
```

Comment:
Can we generify this? We should iterate across the enum constants and find the one that matches, and otherwise throw. Overall in this PR, we should try to minimize the amount of code that needs to change for new nav menu items to as few places as possible.

Reply:
updated to 

```
fun valueFromNavId(id: Int): NavigationDrawerItem {
      for (item: NavigationDrawerItem in values()) {
        if (item.value.equals(id)) return item
      }
      return throw  IllegalArgumentException("NavigationDrawerItem not found by " + id)
    } 
```